### ISSUE TITLE
Prevent Window Resize on App Start

### ISSUE THREAD

**Issue by wyhinton** [fltk-rs/fltk-rs#784]
### Discussed in https://github.com/fltk-rs/fltk-rs/discussions/777

<div type='discussions-op-text'>

<sup>Originally posted by **wyhinton** July 14, 2021</sup>
When creating a new Window, it seems like the size of the window can sometimes resize automatically. I assume that FLTK is trying to account for some kind of screen screen scaling? Or maybe this is a Win10 thing? But on the monitor that I captured the GIF, the screen scale is 1.0. The scale on my other 4k monitor is 1.25. Is there anyway to have the window appear as the correct size before it appears? 

This can also sometimes cause widgets to resize strangely. 

```rust
use fltk::{
    app, app::*, button::*, enums::*, frame::*, group::*, prelude::*, window::*, widget::*, draw::*,
};
//fltk = { version = "1.0.22"}
fn main(){
    let app = App::default();
    let mut win = Window::new(200, 200, 500, 300, "Pack vs Group Overdraw");
    let mut my_pack= Pack::new(50,50,100,200,"mypack");
    let _button= Button::new(0,0,200,20,"mybutton");
    let _button= Button::new(0,0,200,20,"mybutton");
    let _button= Button::new(0,0,200,20,"mybutton");
    let _button= Button::new(0,0,200,20,"mybutton");
    let mut my_group= Group::new(200,50,100,50,"my group");
    my_group.set_frame(FrameType::FlatBox);
    my_group.set_color(Color::Red);


    let mut frame = Frame::new(200,100,100,100, "Test");
    frame.set_frame(FrameType::FlatBox);
    frame.set_color(Color::Yellow);
    my_group.end();

    let mut frame = Frame::new(250,100,100,100, "Test");
    frame.set_frame(FrameType::FlatBox);
    frame.set_color(Color::Yellow);
    my_pack.end();
    win.end();
    win.show();
    app.run().unwrap();
}
```

![window_resize_issue](https://user-images.githubusercontent.com/38958118/125679921-a1579901-5412-4268-bc61-050bd4ebe1ae.gif)

### Demonstrating the odd resizing effect for my DnD ToDoList example
![window_resize_issue2](https://user-images.githubusercontent.com/38958118/125716097-c53094ea-35d6-4d27-8c8a-0844df4dbf65.gif)

### What the window should look like:
![image](https://user-images.githubusercontent.com/38958118/125716171-2d75e7ff-2ba2-4310-9469-db263453a97c.png)

I'm wondering if this might have something to do with the use of ```Pack```. 

</div>

**Comment by wyhinton** [comment]
Looks like I get the same problem when building with C test we used #740 (after adding ```target_link_libraries(main PRIVATE fltk gdiplus)```.

**Comment by MoAlyousef** [comment]
It seem unrelated to gdiplus as well if versions before 1.0.16 also exhibit the same behavior.

**Comment by wyhinton** [comment]
Since we can repro in C maybe I'll just take this to the main fltk repo?

**Comment by MoAlyousef** [comment]
Yeah it's probably better since they might ask some machine and env specific questions.

**Comment by wyhinton** [comment]
Moved to [https://github.com/fltk/fltk/issues/259](https://github.com/fltk/fltk/issues/259)
